### PR TITLE
Update art-unresolved-threads job

### PR DESCRIPTION
**Pull request by ashwindasr**
Changes in this PR:
- Add a new script to notify us in case a certificate is about to expire, within 30 days
- Refactor name to art-notify, since the script is not just showing unresolved threads any more

### CODE REVIEW

**Comment by ashwindasr** [comment]
/hold

Untested. Hoping to do that after team approval

**Review by vfreex — COMMENTED** [review]
/lgtm

This generally looks good to me. After you change the file name, we have to reconfigure Jenkins to point to the new Jenkinsfile locatation.

**Comment by vfreex** [comment]
I saw you created `__init__.py` at the scanning directory. It will make python files under this folder as a whole package. Maybe it makes more sense to move it to the art-notify sub directory?

**Comment by ashwindasr** [comment]
Thanks @vfreex 

Updated in https://github.com/openshift-eng/aos-cd-jobs/pull/4188/commits/2f158319a8a51bd773ab53e71dc522cfe4ec2bda

File: scheduled-jobs/scanning/art-notify/Jenkinsfile [hunk 1]
```diff
@@ -17,6 +17,11 @@ node() {
                         trim: true,
                         defaultValue: "#team-art"
                     ),
+                    booleanParam(
+                        name: 'TAG_RELEASE_ARTISTS',
```
Comment (thegreyd) [inline scheduled-jobs/scanning/art-notify/Jenkinsfile:21]: why add this? if the purpose is not tagging during testing, maybe a dry_run is better

Comment (ashwindasr) [reply scheduled-jobs/scanning/art-notify/Jenkinsfile]: ack - updated

**Pull request merged**